### PR TITLE
Note bug fixes, turn count added

### PR DESCRIPTION
**Pull request by newduke**
* Fixed bug created by HTML parsing notes: "<foo" would not show anything.
* Fixed bug with meaningful notes getting surrounded by brackets: "[k3] some text" -> "[[k3] some text] [cm]"
* Added automated turn count to notes with ctrl + alt + right-click

### CODE REVIEW

File: packages/data/src/abbreviations.ts [hunk 1]
```diff
@@ -21,7 +21,7 @@ export const CHOP_MOVED_NOTES = [
 export const FINESSED_NOTES = ["f", "hf", "pf", "gd", "utf"];
 export const NEEDS_FIX_NOTES = ["fix", "fixme", "needs fix"];
 export const BLANK_NOTES = ["blank"];
-export const CLUED_NOTES = ["clued"];
+export const CLUED_NOTES = ["clued", "y"];
```
Comment (Zamiell) [inline packages/data/src/abbreviations.ts:24]: y is short for yellow so this seems like a bug?

**Comment by Zamiell** [comment]
> Added automated turn count to notes with ctrl + alt + right-click

that seems like a really good feature

File: packages/data/src/abbreviations.ts (hunk 1 above)
Comment (newduke) [reply packages/data/src/abbreviations.ts:24]: Convenience? :)

File: .gitignore [hunk 2]
```diff
@@ -30,6 +30,10 @@ motd.txt*
 /logs
 /ongoing_tables
 
+# Development files
+*.code-workspace
+package-lock.json
```
Comment (Zamiell) [inline .gitignore:35]: add this is a bug because we use yarn, right?

Comment (newduke) [reply .gitignore:35]: I run the build as instructed for linux.  had to `npm install` first to set the lock file.

Comment (Zamiell) [reply .gitignore:35]: thats old info, you shouldn't ever have to invoke npm

Comment (Zamiell) [reply .gitignore:35]: on this page it doesn't say anything about doing `npm install` https://github.com/Hanabi-Live/hanabi-live/blob/main/docs/INSTALL.md

Comment (newduke) [reply .gitignore:35]: That being said, it did not work without first running `npm install` because it complained about a missing lockfile

Comment (Zamiell) [reply .gitignore:35]: its because of this bug https://github.com/Hanabi-Live/hanabi-live/commit/37631cb77966973327eb904d113dce2df4fe1083

**Comment by Zamiell** [comment]
should be live now, thanks

**Comment by Zamiell** [comment]
oh, you should submit a pr that documents the new changes

**Comment by Zamiell** [comment]
on features.md or whatever

**Comment by newduke** [comment]
Wow that's fast. Does it take the bundles from CI then?

**Comment by Zamiell** [comment]
what?

**Comment by newduke** [comment]
Where does the live site get the javascript?  From gh ci?

**Comment by Zamiell** [comment]
no i type git pull

**Pull request merged**